internal/schema/schema_test.go

```diff
@@ -864,6 +867,11 @@ Second line of the description.
 				if len(unionDirectives) != 1 || unionDirectives[0].Name.Name != "uniondirective" {
 					return fmt.Errorf("missing directive on Union union, expected @uniondirective but got %v", unionDirectives)
 				}
+
+				repeatableDirectives := s.Types["Repeatable"].(*types.ScalarTypeDefinition).Directives
+				if len(repeatableDirectives) != 2 || repeatableDirectives[0].Name.Name != "repeatabledirective" || repeatableDirectives[1].Name.Name != "repeatabledirective" {
+					return fmt.Errorf("missing directive on Repeatable scalar, expected @repeatabledirective @repeatabledirective but got %v", repeatableDirectives)
+				}
```

Comment:
In your test you do not verify that the `Rpeatable` property of your directive is set to `true`. Please, test if it is set properly.

Reply:
😬 Thanks, yep, I was not setting that field. I wrote a separate test for this because it didn't feel like it fit in with this test, but I'll combine it if you prefer!